templates/index.html

```diff
@@ -83,12 +84,15 @@ <h2 style="color: white; text-align: center;">COVID-19 Vaccine Finder</h2>
           • Tip: Quickly register for vaccines from the above link after getting all the information you need from our website.
         </p>
       </section>
+      </main>
       <!--Footer-->
       <footer class="ftr-position">
+        <div>
```

Comment:
Please follow the proper Indentation, use a linter & prettifier.

Reply:
Ok sure. but I had to change some of the code from the ground up. 
Instead of using the `position` property, I found that `flex` property gives a more uniform result on multiple screens. this in tend affects some of the spacing. 
But I'll work on a fix as soon as possible